eslzArm/eslz-portal.json

```diff
@@ -2771,6 +2727,205 @@
+                                    "defaultValue": "Yes (recommended)",
+                                    "visible": true,
+                                    "toolTip": "If 'Yes' is selected then Azure Policy will prevent PaaS resources to use public endpoints.",
+                                    "constraints": {
+                                        "allowedValues": [
+                                            {
+                                                "label": "Yes (recommended)",
+                                                "value": "Yes"
+                                            },
+                                            {
+                                                "label": "Audit only",
+                                                "value": "Audit"
+                                            },
+                                            {
+                                                "label": "No",
+                                                "value": "No"
+                                            }
+                                        ]
+                                    }
+                                },
+                                {
+                                    "name": "denyPipOnNicForCorp",
+                                    "type": "Microsoft.Common.OptionsGroup",
+                                    "label": "Prevent usage of NICs with public IP(s) in the corp connected landing zones",
+                                    "defaultValue": "Yes (recommended)",
+                                    "visible": true,
+                                    "toolTip": "If 'Yes' is selected then Azure Policy will prevent NICs with public IPs(s).",
+                                    "constraints": {
+                                        "allowedValues": [
+                                            {
+                                                "label": "Yes (recommended)",
+                                                "value": "Yes"
+                                            },
+                                            {
+                                                "label": "No",
+                                                "value": "No"
+                                            }
+                                        ]
+                                    }
+                                },
+                                {
+                                    "name": "denyHybridNetworking",
+                                    "type": "Microsoft.Common.OptionsGroup",
+                                    "label": "Deny the deployment of vWAN/VPN/ER Gateways to the Corp Management Group?",
+                                    "defaultValue": "Yes (recommended)",
+                                    "toolTip": "If 'Yes' is selected for Corp, vWAN/VPN/ER Gateways will be blocked from deployment to the Corp Management Group",
```

Comment:
```suggestion
                                    "toolTip": "If 'Yes' is selected for Corp, vWAN/VPN/ERs, including Gateways, will be audited if deployed into Subscriptions in the Corp Management Group",
```

Reply:
This assignment DENY's deployment of VWAN/ER/VPN gateways... not sure I understand why you're suggesting "audit"?